README.md

```diff
@@ -1,4 +1,59 @@
-# Convert Obsidian files to org-roam
+# obsidian-to-org
+
+Convert Obsidian files to org-roam
 
 Based on @rberaldo's [pioneering work](https://gist.github.com/rberaldo/2a3bd82d5ed4bc39fee7e8ff4a6242b2).
-See also https://org-roam.discourse.group/t/fully-migrating-from-obsidian/1708
+
+See also
+
+https://org-roam.discourse.group/t/fully-migrating-from-obsidian/1708
+
+https://emacs-china.org/t/obsidian-to-org-python/23125
+
+## Use without install as python package
+
+```shell
+cd obsidian-to-org/src/obsidian-to-org/
+python3 __main__.py [obsidian dir] [output dir]
+```
+
+## Install as python package
+
+Install requirements/dependencies
+
+```shell
+cd obsidian-to-org/
+poetry install
+```
+
+(Optional) Build obsidian-to-org if dist/ doesn't exists
+
+```shell
+poetry build
```

Comment:
Instead of building you can actually do `poetry run obsidian-to-ord-roam source-path output-path`. The `pyproject.toml` file has a `tool.poetry.scripts` section that defines which script name will run which function. :D

Reply:
Yeah, I learned that a well after this PR.

Now I's using org-roam and gave up converting from obsidian to org-roam.